cgmes-gl/cgmes-gl-iidm-extensions/src/main/java/com/powsybl/cgmes/iidm/extensions/gl/Coordinate.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.iidm.extensions.gl;
+
+/**
+ * <a href="https://en.wikipedia.org/wiki/World_Geodetic_System">WGS84</a> coordinate.
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.eu>
+ */
+public class Coordinate {
+
+    private final double lng;
```

Comment:
Why not being more explicit with "longitude" and "latitude" ?
I personally find it easier to read :)

Reply:
Agreed, done.